src/data/opposite.lean

```diff
@@ -55,6 +56,17 @@ lemma unop_inj : function.injective (unop : αᵒᵖ → α) := λ _ _, id
 
 attribute [irreducible] opposite
 
+def equiv_to_opposite : α ≃ αᵒᵖ :=
+{ to_fun := op,
+  inv_fun := unop,
+  left_inv := λ a, by simp,
+  right_inv := λ a, by simp, }
```

Comment:
BTW, with this `equiv` you can get `op_eq_iff_eq_unop` etc from `equiv.eq_symm_apply` etc.